__tests__/order/order.variables.js

```diff
@@ -0,0 +1,395 @@
+const newOrder = {
+  "status":"pending",
```

Comment:
Please, remove quotes for keys of object and replace quotes in values from double to single one

Reply:
fixed